app/src/main/java/com/dsu/poc/repository/login/ValidationRepository.kt

```diff
@@ -0,0 +1,54 @@
+package com.dsu.poc.repository.login
+
+import android.app.Application
+import android.text.TextUtils
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+
+class ValidationRepository(application: Application) {
```

Comment:
why do we need application as parameter here?????????? Never pass Application instance as parameter

Reply:
done